types/eip4844/state.yaml

```diff
@@ -0,0 +1,88 @@
+EIP4844:
+  BeaconState:
+    type: object
+    description: "The [`BeaconState`](https://github.com/ethereum/consensus-specs/blob/master/specs/eip4844/beacon-chain.md#beaconstate) object from the Eth2.0 EIP-4844 spec."
```

Comment:
link will need to update to `/deneb/` rather than `/eip4844`

Reply:
I think you updated all of them in the other PR! :)